ДЗ 13 спринта:

ZarubaGit opened the pull request:
1. Обновил модель трека
2. Создал экран "Аудиоплеера"
3. Реализовал логику перехода с экрана поиска
4. Реализовал логику наполнения информацией экрана Аудиоплеера

ZarubaGit committed:
ДЗ 13 спринта:
1. Обновил модель трека
2. Создал экран "Аудиоплеера"
3. Реализовал логику перехода с экрана поиска
4. Реализовал логику наполнения информацией экрана Аудиоплеера

Charmandik commented on app/src/main/java/com/example/playlistmaker/AudioPlayer.kt:
```diff
@@ -0,0 +1,86 @@
+
+class AudioPlayer : AppCompatActivity() {
+
+    private lateinit var backButton: ImageView
```
Попробуй применить подход View Binding. ViewBinding позволяет проще писать код для взаимодействия с версткой позволяет избежать этого процесса с инициализацией каждой view по отдельности. Более подробно можно почитать в статье https://habr.com/ru/articles/467295/

Charmandik commented on app/src/main/res/layout/activity_audio_player.xml:
```diff
@@ -0,0 +1,251 @@
+        android:layout_height="wrap_content"/>
+
+    <ImageView
+        android:id="@+id/backInSearch"
```
Для более простого использования различных классов view рекомендую использовать следующий трюк: К имени переменной добавлять префикс в виде "аббревиатуры" класса, к которой она относится, для TextView - tv, ImageView - iv, RecyclerView - rv и т.д.. Например, у тебя есть RecyclerView для треков, в этом случае будет rvTracks

ZarubaGit committed:
1. Смена findViewById на ViewBinding в некоторых классах.
2. Мелкая доработка интерфейса: содержимое "Альбом" на экране Аудиоплеера съезжало, если текста было много.
